### PR TITLE
Improve Dag Deploy enable text

### PR DESCRIPTION
**Pull request by kushalmalani**
## Description

> Describe the purpose of this pull request.

Improve Dag Deploy enable text

## 🎟 Issue(s)

Related #XXX

## 🧪 Functional Testing

> List the functional testing steps to confirm this feature or fix.

## 📸 Screenshots

> Add screenshots to illustrate the validity of these changes.

## 📋 Checklist

- [ ] Rebased from the main (or release if patching) branch (before testing)
- [ ] Ran `make test` before taking out of draft
- [ ] Ran `make lint` before taking out of draft
- [ ] Added/updated applicable tests
- [ ] Tested against [Astro-API](https://github.com/astronomer/astro/) (if necessary).
- [ ] Tested against [Houston-API](https://github.com/astronomer/houston-api/) and [Astronomer](https://github.com/astronomer/astronomer/) (if necessary).
- [ ] Communicated to/tagged owners of respective clients potentially impacted by these changes.
- [ ] Updated any related [documentation](https://github.com/astronomer/docs/)

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-*       @andriisoldatenko @aliotta @neel-astro @AmFlint @kushalmalani @sunkickr
+*       @andriisoldatenko @aliotta @neel-astro @AmFlint @kushalmalani @sunkickr @jemishp
```
Comment (kushalmalani) [inline CODEOWNERS:1]: Adding @jemishp to code owners

File: cmd/cloud/deployment.go [hunk 2]
```diff
@@ -307,8 +307,8 @@ func deploymentUpdate(cmd *cobra.Command, args []string) error {
 		return errors.Wrap(err, "failed to find a valid workspace")
 	}
 
-	if dagDeployEnable != "" && !(dagDeployEnable == enable || dagDeployEnable == disable) {
-		return errors.New("Invalid --dag-deploy value. Possible values are (enable, disable)")
+	if dagDeploy != "" && !(dagDeploy == enable || dagDeploy == disable) {
+		return errors.New("Invalid --dag-deploy value)")
```
Comment (jemishp) [inline cmd/cloud/deployment.go:311]: I would add missing test cases for this so we have better test coverage.

Comment (neel-astro) [inline cmd/cloud/deployment.go:311]: nitpick: might want to add unit tests for this case

**Review by neel-astro — APPROVED** [review]
LGTM 👍 , have left a minor nitpick

**Review by jemishp — APPROVED** [review]
I would bump up test coverage and I can help with it. 

But other than that looks good.

File: cmd/cloud/deployment.go (hunk 2 above)
Comment (kushalmalani) [reply cmd/cloud/deployment.go:315]: Done

Comment (kushalmalani) [reply cmd/cloud/deployment.go:315]: Done

**Pull request merged**